diagnostic.md

```diff
@@ -5,74 +5,84 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+// The back end stores data that is accessed by the user indirectly through a server allows developer to control format and flow of information from client.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+// server
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+// controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+//because the controller should effect views not the developer
```

Comment:
`Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.`